src/test/regress/expected/failure_split_cleanup.out

```diff
@@ -412,25 +412,25 @@ CONTEXT:  while executing command on localhost:xxxxx
 
     -- Left over publications
     SELECT pubname FROM pg_publication;
-              pubname
+                pubname
 ---------------------------------------------------------------------
- citus_shard_split_publication_xxxxxxx_xxxxxxx
- citus_shard_split_publication_xxxxxxx_xxxxxxx
+ citus_shard_split_publication_xxxxxxx_xxxxxxx_777
+ citus_shard_split_publication_xxxxxxx_xxxxxxx_777
 (2 rows)
 
     -- Left over replication slots
     SELECT slot_name FROM pg_replication_slots;
-          slot_name
+           slot_name
 ---------------------------------------------------------------------
- citus_shard_split_slot_xxxxxxx_xxxxxxx
- citus_shard_split_slot_xxxxxxx_xxxxxxx
+ citus_shard_split_slot_xxxxxxx_xxxxxxx_0
+ citus_shard_split_slot_xxxxxxx_xxxxxxx_0
 (2 rows)
 
     -- Left over subscriptions
     SELECT subname FROM pg_subscription;
-              subname
+                subname
 ---------------------------------------------------------------------
- citus_shard_split_subscription_xxxxxxx
+ citus_shard_split_subscription_xxxxxxx_777
```

Comment:
having these numbers in there is probably causing issues when running the test with `check-minimal`. You should update the relevant rules in the `normalize.sed` file.